### PR TITLE
[th] Translated Chapter2/1

### PR DESCRIPTION
**Pull request by ckingkan**
Hi, Chapter2/1 has been translated to [th] #64. Please let me know if there is anything need changes.

### CODE REVIEW

File: chapters/th/_toctree.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- title: 1. โมเดล Transformer
+  sections:
+  - local: chapter1/1
+    title: บทนำ
+
+- title: 2. การใช้ 🤗 Transformers
+  sections:
+  - local: chapter2/1
+    title: บทนำ
+  - local: chapter2/2
```
Comment (lewtun) [inline chapters/th/_toctree.yml:15]: We should only include the chapters / sections in the ToC for content that has already been translated. So in this case you can remove `chapter2/2` and Chapter 3 :)

**Review by lewtun — APPROVED** [review]
Thank you for this contribution @ckingkan 🚀 ! 

You just need to resolve a merge conflict with the `_toctree.yml` file, but otherwise this looks great!

File: chapters/th/_toctree.yml (hunk 1 above)
Comment (lewtun) [reply chapters/th/_toctree.yml:15]: Hi @ckingkan would you still like to work on this last change or would you prefer that I take care of it (happy to do so!)

Comment (ckingkan) [reply chapters/th/_toctree.yml]: Hi @lewtun, sorry for my late reply. I have fixed the conflicts

**Pull request merged**